cmake/CMakeLists.txt

```diff
@@ -211,12 +212,6 @@ if (MSVC)
     set(protobuf_MSVC_STATIC_RUNTIME OFF CACHE BOOL "Link protobuf to static runtime libraries" FORCE)
     set(gtest_force_shared_crt ON CACHE BOOL "Use shared (DLL) run-time lib for gtest" FORCE)
   endif()
-  #A compile option to improve code generation time, mostly for the code that heavily uses eigen.
-  check_cxx_compiler_flag(/d2ReducedOptimizeHugeFunctions HAS_D2ROHF)
```

Comment:
Does /d2ReducedOptimizeHugeFunctions conflict with -DEIGEN_STRONG_INLINE?

Reply:
No. BUT only one of them is needed.

Reply:
d2ReducedOptimizeHugeFunctions requires vs 2019